### DISCUSSION TITLE
SQLClient adds more than 100 new child dependencies

### DISCUSSION THREAD

**Discussion by Rasic10** [dotnet/SqlClient#3054] [Q&A] [answered]
I'm in the process of migrating System.Data.SqlClient to Microsoft.Data.SqlClient and I installed the last stable version 5.2.2 in my projects but realized that NuGet package has a lot of child dependencies that were added (more then 100)
Is there any solution to avoid it or any idea about it?

**Comment by David-Engel** [comment, marked as answer]
See #1108 for reference. There are some options discussed there to strip out unused dependencies and we have plans to split out the large Azure-related dependencies in a future release.
